app/models/exercise/reading.rb

```diff
@@ -11,6 +11,10 @@ def input_kids?
     false
   end
 
+  def layout=(layout)
```

Comment:
Nice! We should make a list of layouts that are coupled to exercise types (if there are more) and generalize this one.

Reply:
Yup. I want to remove this restriction instead of just documenting, actually